Shared/Dockpane1.xaml

```diff
@@ -75,13 +75,18 @@
             <RowDefinition Height="8" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <ListBox Grid.Row="0" x:Name="MainTabs" Style="{DynamicResource PanelIndicator}" SelectedIndex="0" >
-            <ListBox.Items>
-                <frameworkControls:PanelHeader Text="File"/>
-                <frameworkControls:PanelHeader Text="Fields"/>
-            </ListBox.Items>
-        </ListBox>
-        <Grid Grid.Row="2" x:Name="FileGrid" ScrollViewer.HorizontalScrollBarVisibility="Disabled" ScrollViewer.VerticalScrollBarVisibility="Auto">
+        <DockPanel Grid.Row="0" LastChildFill="True">
+            <Button DockPanel.Dock="Right" x:Name="Refresh_Button" VerticalAlignment="Center" Style="{StaticResource Button16}" Click="MethodPanelrefreshXML_Click" ToolTip="Refresh the Source-Target Configuration File">
+                <Image Source="pack://application:,,,/ArcGIS.Desktop.Resources;component/Images/GenericRefresh16.png"></Image>
+            </Button>
+            <ListBox DockPanel.Dock="Left" x:Name="MainTabs" Style="{DynamicResource PanelIndicator}" SelectedIndex="0" >
+                <ListBox.Items>
+                    <frameworkControls:PanelHeader Text="File"/>
+                    <frameworkControls:PanelHeader Text="Fields"/>
+                </ListBox.Items>
+            </ListBox>
+        </DockPanel>
+        <Grid Grid.Row="3" x:Name="FileGrid" ScrollViewer.HorizontalScrollBarVisibility="Disabled" ScrollViewer.VerticalScrollBarVisibility="Auto">
```

Comment:
This grid and the FieldsGrid should remain on Grid.Row="2" since we didn't add a new row.